### PR TITLE
Add community-specific documentation: DMCC

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

Add community-specific documentation: DMCC

---

**Related issue :**

### CODE REVIEW

**Review by glarocca — APPROVED** [review]
Approve commit

File: content/en/users/communities/dmcc/_index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: "Disaster Mitigation and Agriculture community"
+linkTitle: "Disaster Mitigation and Agriculture community"
+type: docs
+weight: 10
+description: >
+  How to to use the EGI infrastructure for Disaster Mitigation and Agriculture community
+---
+
+This is the documentation to suppor the Disaster Mitigation and Agriculture community.
```
Comment (gwarf) [inline content/en/users/communities/dmcc/_index.md:10]:
```suggestion
This is the documentation to support the Disaster Mitigation and Agriculture community.
```

File: content/en/users/communities/dmcc/_index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Researchers
+
+### How to get access
+
+Getting access to the EGI infrastructure consist of the following steps:
```
Comment (gwarf) [inline content/en/users/communities/dmcc/_index.md:16]:
```suggestion
Getting access to the EGI infrastructure consists of the following steps:
```

File: content/en/users/communities/dmcc/_index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+   * Network ID: `group-project-network`
+   * Security Group: ingress traffic to port 22 (SSH) is enabled
+
+### How to bring your scientific application to the EGI infrastructure
+
+The official solution to distribute software in the EGI Infrastructure
+is to use [CVMFS](https://cernvm.cern.ch/fs/), the Software
+Distribution Service developed to assist High Energy Physics collaborations
+at CERN. For more details, please refer to the
+[CVMFS](../../compute/content-distribution/) documentation.
```
Comment (gwarf) [inline content/en/users/communities/dmcc/_index.md:38]:
```suggestion
[Contend Distribution](../../compute/content-distribution/) documentation.
```
I think we should promote the name of the service `Content Distribution` more than the technology "currently" used to implement it.

File: content/en/users/communities/dmcc/_index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+The following alternative solutions to share scientific software in the
+EGI Infrastructure are also possible:
```
Comment (gwarf) [inline content/en/users/communities/dmcc/_index.md:41]:
```suggestion
The following alternative solutions for sharing scientific software in the
EGI Infrastructure are also available:
```

File: content/en/users/communities/dmcc/_index.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+* Use the [EGI Applications Database](https://appdb.egi.eu/) (AppDB):
+  either [packaging your application in a custom Virtual Machine](../../compute/cloud-compute/images/)
+  or [uploading a Virtual Appliance](https://wiki.appdb.egi.eu/main:faq:how_to_register_a_virtual_appliance).
+* Use [Docker](https://www.docker.com/) containers.
+
+### How to bring your data to the EGI infrastructure
+
+Different Data Management services are available in the EGI Infrastructure for dealing with data:
```
Comment (gwarf) [inline content/en/users/communities/dmcc/_index.md:50]:
```suggestion
For managing data, different Data Management services are available in the EGI Infrastructure:
```

File: content/en/users/communities/dmcc/_index.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+* [EGI Data Orchestrator](../../data/management/rucio/)
+* [OpenRDM](../../data/management/open-rdm/)
+
+For more information, please see the [Data Management section](../../data/management/).
+
+## Service Providers
+
+### How to contribute cloud resources
+
+The steps that an OpenStack cloud provider needs to follow to add resources
```
Comment (gwarf) [inline content/en/users/communities/dmcc/_index.md:63]:
```suggestion
The steps that an OpenStack cloud provider need to follow to add resources
```

**Review by gwarf — CHANGES_REQUESTED** [review]
LGTM, just suggested a few typos/redirection.
For the record, in the past I already prepared some community-specific documentation, and I feel that it's indeed something that can be very valuable for less advanced users, or just saving users time for figuring (and potentially documenting) this themselves. Glad to see this!

File: content/en/users/communities/dmcc/_index.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+* [EGI Data Orchestrator](../../data/management/rucio/)
+* [OpenRDM](../../data/management/open-rdm/)
+
+For more information, please see the [Data Management section](../../data/management/).
+
+## Service Providers
+
+### How to contribute cloud resources
+
+The steps that an OpenStack cloud provider need to follow to add resources
```
Comment (gwarf) [inline content/en/users/communities/dmcc/_index.md:63]:
```suggestion
The steps that an OpenStack cloud provider needs to follow to add resources
```
Duh, sorry for this error, I will merge this fix myself, you were right :)

**Review by gwarf — APPROVED** [review]

**Pull request merged**